virttest/bootstrap.py

```diff
@@ -955,7 +956,6 @@ def bootstrap(options, interactive=False):
                 LOG.debug("Module %s loaded", module)
 
     LOG.info("")
-    LOG.info("VT-BOOTSTRAP FINISHED")
-    LOG.debug("You may take a look at the following online docs for more info:")
-    LOG.debug(" - http://avocado-vt.readthedocs.org/")
-    LOG.debug(" - http://avocado-framework.readthedocs.org/")
+    LOG.info("vt-bootstrap finished")
```

Comment:
Yes, I actually thought we can skip 52.0 for Avocado-VT.  Not that it would hurt, but since 52.0 is Avocado's LTS, and Avocado-VT doesn't have such differenciated release, I left it without a tag/release.

Now back to the feature.  I'm fine with trying to come with a backward compatible change, but it looks like what you're proposing here is actually different from what I intended.  Not that my idea is the absolute one, but this change brings another round of "incompatible" changes.  Let me explain.

As it stands, `--vt-no-downloads` will skip *any* download operation, no matter the bootstrap phase.  Then, option `--vt-skip-verify-download-assets` can also let users choose to skip that individual phase.  

My idea was to expand that to other phases, so other `--vt-skip-<PHASE>` could be added if those made sense. Those phases may or may not involve downloading content, so `--vt-no-downloads` definitely has a different meaning.

Question is: do you want to revert the behavior, the option names, or both?  Given the difference in the expected end result, I don't see that an alias like you're proposing would do.